sms_services/kcell.rb

```diff
@@ -1,48 +0,0 @@
-module Services::Sms
-  class Kcell < Base
-
-    def send_request!(params)
-      uri = URI(url)
-
-      http = Net::HTTP.new(uri.host)
-      request = Net::HTTP::Post.new(uri.request_uri, headers)
-      request.basic_auth(config["username"], config["password"])
-      request.body = params.to_json
-      http.request(request)
-    end
-    
-
-    def self.phone_normalization_klass
-      PhoneNumbers::Kcell
-    end
-
-    protected
-
-    def url
-      "#{ super }/messages"
```

Comment:
With interpolated expressions, there should be no padded-spacing inside the braces.
https://github.com/rubocop-hq/ruby-style-guide#spaces-and-braces